src/main/resources/logback-spring.xml

```diff
@@ -53,10 +52,6 @@
     <appender-ref ref="logAppender"/>
```

Comment:
If you are removing the logAppender, this config will not work.
You can just log everything to the console if you like I think.
See this thread and linked PR for more details of how to approach this

 https://mojdt.slack.com/archives/C69NWE339/p1681458769482799

Reply:
Thank you.

I read the https://learn.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-upgrade-from-2x#step-1-update-dependencies 

migration documentation and it said

> Remove the dependency and remove the Application Insights appender from your Logback configuration. 
> No longer needed since Logback is autoinstrumented in the 3.x Java agent.

-----

It was using `com.microsoft.applicationinsights.logback` in the logback-spring.xml file

An error would occur if i added the appender back in 🤔 

> Logback configuration error detected:
  ERROR in ch.qos.logback.core.model.processor.AppenderModelHandler - Could not create an Appender of type [com.microsoft.applicationinsights.logback.ApplicationInsightsAppender]. ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type com.microsoft.applicationinsights.logback.ApplicationInsightsAppender 


I wasn't sure what to do here so i thought, merge this in to PREPROD and test application insights

Reply:
More here https://learn.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-config#autocollected-logging

I think it defaults to INFO but we can set the log level to ERROR in applicationinsights.json

Reply:
I don't think my comment was very clear, sorry..

What I mean is that the removal of the logAppender on line 25 (which is correct as you say, it will give an error otherwise) means that in the rest of the config, we cannot use that appender as it doesn't exist

So all the lines like this:
`<appender-ref ref="logAppender"/>`

Need to be replaced by:
`<appender-ref ref="consoleAppender"/>`

Reply:
Let's talk about it later when we pair :-)